src/app/main-page/main-page.component.html

```diff
@@ -0,0 +1,23 @@
+<h1>My Projects</h1>
+<div class="project-container">
+  <div class="project">
+    <img
+      class="pic-pokemon"
+      src="https://sm.ign.com/t/ign_il/screenshot/default/c89b6bc7-0673-4720-b761-bab17c7d53fa-xoq7fo_119x.1280.jpg"
```

Comment:
usually we are create folder called assets --> put all the icons/images 

i want you find out more way for import this image , without writing directly all the url in the html

Reply:
fixed